namespaces/opts_linux.go

```diff
@@ -14,17 +14,16 @@
    limitations under the License.
 */
 
-package containerd
+package namespaces
 
 import (
 	"context"
 
 	"github.com/containerd/cgroups"
-	"github.com/containerd/containerd/namespaces"
 )
 
 // WithNamespaceCgroupDeletion removes the cgroup directory that was created for the namespace
-func WithNamespaceCgroupDeletion(ctx context.Context, i *namespaces.DeleteInfo) error {
+func WithNamespaceCgroupDeletion(ctx context.Context, i *DeleteInfo) error {
```

Comment:
could be just `WithCgroupDeletion` now

Reply:
Except there's a `WithNamespace` too so wanted to be consistent.

Reply:
But I'm happy to change things if maintainers agree

Reply:
Well, something was broken by this in the first place. It should never have used the `namespaces` package. I will look into correcting this.

Reply:
I agree `WithCgroupDeletion` is a better name. The `WithNamespace` is a specific pattern for contexts, not for namespace options.